astroquery/linelists/cdms/core.py

```diff
@@ -317,6 +327,8 @@ def get_species_table(self, *, catfile='partfunc.cat', use_cached=True,
         """
 
         if use_cached:
+            if write_new_species_cache:
+                warnings.warn("use_cached and write_new_species_cache are both set to True; write_new_species_cache will be ignored.  If you meant to update the cache, set use_cached to False.")
```

Comment:
This now issues a couple of warnings in the tests that we need to handle.

Reply:
Uhoh.... that shouldn't be happening.